crates/transaction-emitter-lib/src/transaction_generator/p2p_transaction_generator.rs

```diff
@@ -50,12 +48,10 @@ impl P2PTransactionGenerator {
         to: &AccountAddress,
         num_coins: u64,
         txn_factory: &TransactionFactory,
-        gas_price: u64,
     ) -> SignedTransaction {
         from.sign_with_transaction_builder(
             txn_factory
-                .payload(aptos_stdlib::aptos_coin_transfer(*to, num_coins))
-                .gas_unit_price(gas_price),
+                .payload(aptos_stdlib::aptos_coin_transfer(*to, num_coins)),
```

Comment:
Silly question: why did we remove the `gas_price`?

Reply:
gas_price was redundant here. it is already set on the txn_factory that is passed to the generator, no need to pass it again and set it again (to the same value)